src/main/resources/templates/projects/list.html

```diff
@@ -1,67 +1,94 @@
-          data-th-each="author : ${project.authors}"
-        >
-          <a
-            href="users/public.html"
-            data-th-href="@{/user/{id}(id=${author.userId})}"
-            data-th-text="${author.name}"
-          >Author Name</a><span
-            data-th-if="not ${authorStat.last}"
-          >, </span>
-        </span>
-        <ul class="global-project-list-item-tag-list">
+  <head>
+    <title>Recurse Center Community Portfolio</title>
+    <link rel="stylesheet" href="/style.css" />
+  </head>
+  <body>
+    <main>
+      <h2>Project List</h2>
+      <section>
+        <ul class="global-project-list" data-th-fragment="list(projects)">
           <li
-            class="global-project-list-item-tag-list-item"
-            data-th-each="tag : ${project.tags}"
+            class="global-project-list-item"
+            data-th-each="project : ${projects}"
           >
-            <a
-              class="global-project-list-item-tag-list-item-link"
-              href="../tags/tag.html"
-              data-th-href="@{/tag/{id}(id=${tag.tagId})}"
-              data-th-text="${tag.name}"
-            >
-              Tag Name
-            </a>
+            <div class="global-list-left-column">
+              <div class="global-list-author-images">
+                <span
+                  data-th-each="author : ${project.authors}"
+                  class="badge-image-container"
+                >
+                  <a data-th-href="@{/user/{id}(id=${author.userId})}">
+                    <img
+                      class="badge-image"
+                      src="../../static/user-placeholder.png"
+                      data-th-src="${author.imageUrl}"
+                  /></a>
+                </span>
+              </div>
+              <div class="global-list-project-title-column">
+                <div class="description-project-name">
+                  <a
+                    href="projects/public.html"
+                    data-th-href="@{/project/{id}(id=${project.projectId})}"
+                    data-th-text="${project.name}"
+                    >Project Name</a
+                  >
+                </div>
+                <div class="global-list-authors-list">
+                  <span
+                    class="global-project-author"
+                    data-th-each="author : ${project.authors}"
+                  >
+                    <div class="description-container">
+                      <div>
+                        <a
+                          class="global-project-list-item-author"
+                          href="users/public.html"
+                          data-th-href="@{/user/{id}(id=${author.userId})}"
+                          data-th-text="${author.name}"
+                          >Author Name</a
+                        ><span
+                          style="margin: 0 0.1em"
+                          data-th-if="not ${authorStat.last}"
+                          >&middot</span
```

Comment:
I like how this looks!

Also, TIL why this character entity works without a ending semicolon: https://stackoverflow.com/a/15532395/ . Neat!

I think I might later replace this HTML implementation with a [CSS-only solution](https://stackoverflow.com/a/11574821/), but that doesn't need to be part of this PR - mostly this is a note-to-self, with a little bit of "you might be interested in...". :)